ms-common-impl/src/main/java/net/trajano/ms/vertx/jaxrs/JwtAssertionInterceptor.java

```diff
@@ -1,13 +1,21 @@
 package net.trajano.ms.vertx.jaxrs;
 
-import java.io.IOException;
-import java.net.URI;
-import java.text.ParseException;
-import java.util.concurrent.ExecutionException;
+import com.google.common.cache.CacheBuilder;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.google.common.cache.CacheBuilder'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)